### DISCUSSION TITLE
How to download image/document

### DISCUSSION THREAD

**Discussion by vishalg2235** [open-wa/wa-automate-nodejs#2493] [Q&A] [answered]
Hello,
Im stuck at a point where I want to get the media sent via Whatsapp. Media can be image, document, video, audio etc. So its responding with following Message object. But how can I get the media file?

**Comment by smashah** [comment, marked as answer]
No need to doxx yourself.

Answer is in the docs.

https://docs.openwa.dev/pages/How%20to/decrypt-media.html

**Reply by vishalg2235** [reply]
oh Thannk @smashah
